### ISSUE TITLE
Failed to download 12.0.1 RPM file (404)

### ISSUE THREAD

**Issue by adborden** [srsp/ansible-oracle-java#29]
When using the latest [master](https://github.com/srsp/ansible-oracle-java/commit/fd61f7b15cce3b475b1ff92c8ec2dcfe1d73772f) version, the role failed to download the RPM file.

Variables I'm setting:

```
  - role: srsp.oracle-java
    java_remove_download: false
```

**Comment by adborden** [comment]
I worked around this by setting:
```
    jdk_file_url: https://download.oracle.com/otn-pub/java/jdk/12.0.2+10/e482c34c86bd4bf8b56c0b35558996b9/jdk-12.0.2_linux-x64_bin
    java_subversion: "0.2"
```

**Comment by hypery2k** [comment]
also Java 11 is broken

**Comment by srsp** [comment]
@adborden I added the vars from your comment to the master branch and tested it with Debian 9. So JDK 12 should work again in the latest version.

**Comment by srsp** [comment]
@hypery2k All JDK 11 versions are now behind the Oracle login. So it is not possible to download this version automatically with this role. You could however download the JDK with your Oracle account and use the role to install it. I described the two possible ways in the readme.

**Comment by srsp** [comment]
Fixed with 3.2.0 for JDK 12. JDK 11 is unfixable.
